dspace/config/dspace.cfg

```diff
@@ -1448,21 +1448,11 @@ sitemap.engineurls = http://www.google.com/webmasters/sitemaps/ping?sitemap=
 #####  SHERPA/Romeo Integration Settings ####
 # the SHERPA/RoMEO endpoint
 sherpa.romeo.url = http://www.sherpa.ac.uk/romeo/api29.php
-
-# to disable the sherpa/romeo integration
-# uncomment the follow line
-# webui.submission.sherparomeo-policy-enabled = false
-
-# please register for a free api access key to get many benefits
-# http://www.sherpa.ac.uk/news/romeoapikeys.htm
-# sherpa.romeo.apikey = YOUR-API-KEY
```

Comment:
@KevinVdV : Another very tiny change.  Could we please revert to using this existing `sherpa.romeo.apikey` and restore the comments about that configuration?   It looks like we've changed the name of this field to `sherpa.romeo.apiKey` (notice the capital "K"), which will mean that existing configurations will no longer work and will need updating.

Reply:
After the above tiny change is made ⬆️ , this PR seems good to me. I don't notice any other issues or concerns.